tests/tests_unit/rules/test_models/test_information_rules.py

```diff
@@ -216,51 +327,15 @@ def test_david_as_dms(self, david_spreadsheet: dict[str, dict[str, Any]]) -> Non
 
         assert isinstance(dms_rules, DMSRules)
 
-    def test_olav_as_dms(self, olav_rules: InformationRules) -> None:
-        olav_rules_copy = olav_rules.copy(deep=True)
-        # Todo: Remove this line when Olav's Information .xlsx file is available
-        new_classes = SheetList[InformationClass](data=[])
-        for cls_ in olav_rules_copy.classes:
-            if cls_.class_.versioned_id == "power_analytics:GeoLocation":
-                continue
-            elif cls_.class_.versioned_id in ("power_analytics:Point", "power_analytics:Polygon"):
-                cls_.parent = None
-            new_classes.append(cls_)
-        olav_rules_copy.classes = new_classes
-        ## End of temporary code
-
-        dms_rules = olav_rules_copy.as_dms_architect_rules()
-
-        assert isinstance(dms_rules, DMSRules)
-        schema = dms_rules.as_schema()
-
-        wind_turbine = next((view for view in schema.views if view.external_id == "WindTurbine"), None)
-        assert wind_turbine is not None
-        expected_containers = {
-            dm.ContainerId("power", "GeneratingUnit"),
-            dm.ContainerId("power", "WindTurbine"),
-            dm.ContainerId("power_analytics", "WindTurbine"),
-        }
-        missing = expected_containers - wind_turbine.referenced_containers()
-        assert not missing, f"Missing containers: {missing}"
-        extra = wind_turbine.referenced_containers() - expected_containers
-        assert not extra, f"Extra containers: {extra}"
-
-        wind_farm = next((view for view in schema.views if view.external_id == "WindFarm"), None)
-        assert wind_farm is not None
-        expected_containers = {dm.ContainerId("power", "EnergyArea"), dm.ContainerId("power_analytics", "WindFarm")}
-        missing = expected_containers - wind_farm.referenced_containers()
-        assert not missing, f"Missing containers: {missing}"
-        extra = wind_farm.referenced_containers() - expected_containers
-        assert not extra, f"Extra containers: {extra}"
+    @pytest.mark.parametrize("rules", list(correct_solution_rules()))
+    def solution_model_build(self, rules: dict[str, dict[str, Any]]) -> None:
+        issues = DataModelingScenarioValidator(InformationRules.model_validate(rules), IssueList()).validate()
+        assert len(issues) == 0
 
-        point = next((view for view in schema.views if view.external_id == "Point"), None)
-        assert point is not None
-        assert point.implements == [dm.ViewId("power", "Point", "0.1.0")]
+        rules["Properties"][1]["Value Type"] = "power:Point2"
```

Comment:
Suggest adding a comment on why this is causing an issue. I assume it is because `power:Point2` is not in the model.

An even better alternative is to write an explicit assertion for the issues below, something like
```python
assert len(issues) == 1
issue = issues[0]
assert issue == InvalidValueTypeError(...)
```